CMakeLists.txt

```diff
@@ -11,6 +11,21 @@ if(NOT ANDROID_ABI)
     set_property(CACHE ANDROID_ABI PROPERTY STRINGS "armeabi" "armeabi-v7a" "armeabi-v7a with NEON" "armeabi-v7a with VFPV3" "armeabi-v6 with VFP" "x86" "mips" "arm64-v8a" "x86_64" "mips64")
 endif()
 
+if(NOT ANDROID_STL)
+    set(ANDROID_STL "gnustl_shared" CACHE STRING "Android STL")
+    set_property(CACHE ANDROID_ABI PROPERTY STRINGS "gnustl_shared")
```

Comment:
This line and ones like it only make sense if you are providing multiple options - it serves to populate the dropdown in the CMake GUI.